### ISSUE TITLE
Missing api_key error when providing registered Vonage phone number

### ISSUE THREAD

**Issue by dwightwatson** [Vonage/vonage-php-sdk-core#373]
I'm using Laravel and the Vonage notification channel it has. In the latest minor release it moved from vonage-core v3 to v4. After that upgrade I am no longer able to send messages using the channel and instead a `Missing api_key` exception is thrown.

```
Vonage\Client\Exception\Request
Vonage\SMS\ExceptionErrorHandler:72
```

I narrowed down code that replicates this error:
```php
$credentials = new Vonage\Client\Credentials\Basic(config('vonage.api_key'), config('vonage.api_secret'));

$vonage = new Vonage\Client($credentials);

$vonage->sms()->send(
    new \Vonage\SMS\Message\SMS('personalPhoneNumber', config('vonage.sms_from'), 'Testing')
);
```

You can ignore the `config()` functions - they simply provide the correct configuration through Laravel. **The value of `vonage.sms_from` is a US phone number I have registered in Vonage**. That phone number works correctly with v3.

Even more curious - if I replace `config('vonage.sms_from')` with any other number (including my own personal phone number) - the code works fine. Something about using this Vonage number is causing the exception to be thrown.

## Expected Behavior
I would expect that the code above would send the text message to my personal number from my registered Vonage phone number.

## Current Behavior
An exception is thrown indicating that the `api_key` is missing, though it is not. It seems there may be a different configuration error or another issue with v4.

## Context
I'm simply trying to upgrade to v4 of this library and have had to revert to v3 in production until I can resolve this.

## Your Environment
* Version used: 4.0.7
* Environment name and version: PHP 8.2

**Comment by dwightwatson** [comment]
I've now tested using a HTTP request (using Laravel's HTTP client) and it works. So the issue doesn't seem to be with my `from` number so much as something in the library is going on.

```php
Http::baseUrl('https://rest.nexmo.com')
        ->post('sms/json', [
            'api_key' => config('vonage.api_key'),
            'api_secret' => config('vonage.api_secret'),
            'from' => config('vonage.sms_from'),
            'to' => 'personalPhoneNumber',
            'text' => 'Testing',
        ]);
```

**Comment by dwightwatson** [comment]
Alright - so turns out the message is sent when I include the + prefix for the number. `+1217111XXXX` works, but `1217111XXXX`. Clearly v3 didn't require the prefix + but now v4 does? 

Would love any clarification on this - either way, having `Missing api_key` as the thrown error is an issue.

**Comment by SecondeJK** [comment]
(I ask because this code looks like you have a Laravel app, that is then using this SDK as a requirement which would explain the code snippet being similar to how you would use this library without any wrappers)

**Comment by dwightwatson** [comment]
I am using `laravel/vonage-notification-channel` but when I discovered this error I was able to replicate it using the Vonage client directly as shown in the code above.

Inexplicably - I ran the exact same code this morning as I did yesterday when reporting this issue and it now works as expected. It now works with and without the `+` prefix. No idea what was causing this then but appears to be resolved.

Apologies for the noise, thanks for your assistance.
